### ISSUE TITLE
Error on installing a new theme via the UI

### ISSUE THREAD

**Issue by ttrowell** [pantheon-systems/drops-8#113]
Reproduced on new spinup:
http://screencast.com/t/sdjF7VMgjS

The website encountered an unexpected error. Please try again later.
Recoverable fatal error: Argument 1 passed to Drupal\Core\Extension\ThemeHandler::addTheme() must be an instance of Drupal\Core\Extension\Extension, null given, called in /srv/bindings/1443efbfc3c64533b3711e3523cabcac/code/core/includes/theme.maintenance.inc on line 96 and defined in Drupal\Core\Extension\ThemeHandler->addTheme() (line 202 of core/lib/Drupal/Core/Extension/ThemeHandler.php).

CASE : 49701

**Comment by greg-1-anderson** [comment]
I thought that perhaps this might just be a duplicate of https://github.com/pantheon-systems/drops-8/issues/114, but I cannot reproduce this, whether I provide the Theme via URL or by uploading the tar.gz file, and whether or not I have the tmp directory correctly set.

Any other thoughts on isolation?

**Comment by greg-1-anderson** [comment]
I wiped my 8.0.0 site, re-installed, and tried to install zurb through the admin interface by providing the http URL.  I got the following error:
```
zurb_foundation-7.x-4.1.tar.gz does not contain any .info.yml files.
```
Curious.

**Comment by greg-1-anderson** [comment]
Oh, grabbed the wrong version!  Let's try that again.

**Comment by greg-1-anderson** [comment]
Okay, I created a completely fresh site, ran the installer, and successfully installed http://ftp.drupal.org/files/projects/zurb_foundation-8.x-5.0-beta1.tar.gz via URL.  Did not make any modifications to the site at all. So, still not sure how to reproduce.

If you have a reproducible case on your end, please fix the temporary directory path per #114, and see if that clears up the problem.

**Comment by ttrowell** [comment]
Hm, broke again with same error:
`The website encountered an unexpected error. Please try again later.
Recoverable fatal error: Argument 1 passed to Drupal\Core\Extension\ThemeHandler::addTheme() must be an instance of Drupal\Core\Extension\Extension, null given, called in /srv/bindings/c0f696b7c4f444b2967e502205d98b58/code/core/includes/theme.maintenance.inc on line 96 and defined in Drupal\Core\Extension\ThemeHandler->addTheme() (line 202 of core/lib/Drupal/Core/Extension/ThemeHandler.php).`

http://screencast.com/t/0DRHp64oK73a

Drush status for the site:
http://screencast.com/t/y8WdNuG6

This shows the /tmp dir at the binding level, which is desired.

Now, given the update to the upstream regarding the /tmp dir, should I still implement #114?

**Comment by greg-1-anderson** [comment]
#114 is already deployed.

**Comment by ttrowell** [comment]
Right, so then there is no need for added code in settings.php to set the correct tmp directory path.

**Comment by greg-1-anderson** [comment]
Yep.
